content/articles/track-idividual-logged-in-users-using-google-analytics/index.md

```diff
@@ -0,0 +1,194 @@
+### Introduction
+Google Analytics is one of the most popular digital analytics software.
+It is Google’s free web analytics service that allows you to analyze in-depth detail about the visitors to your website.
+It provides valuable insights that can help you to shape the success strategy of your business.

```

Comment:
A more briefer intro would be better.

Have a look at other articles to get an idea.

Reply:
@Nancy245 Don't think this has been done yet?